.sync.yml

```diff
@@ -39,10 +39,18 @@ Gemfile:
           - mswin
           - mingw
           - x64_mingw
+      - gem: 'puppet_litmus'
+        platforms:
+        - ruby
+        - mswin
+        - mingw
+        - x64_mingw
+        condition: "ENV['PUPPET_GEM_VERSION'].nil? or ENV['PUPPET_GEM_VERSION'] !~ %r{ 5}"
```

Comment:
Hi @lionce we can remove the puppet_litmus gem from sync.yaml file.
See below the sample pr https://github.com/puppetlabs/puppetlabs-motd/pull/231

Reply:
yes, sure